docs/README.md

```diff
@@ -40,28 +47,17 @@ This is the website that is live at [gitcoin.co](https://gitcoin.co)
   * [Integrating the 'available work widget' on your repo.](#integrating-the--available-work-widget--on-your-repo)
   * [Adding GitcoinBot to your repo](#adding-gitcoinbot-to-your-repo)
   * [Adding your token to Gitcoin](#adding-your-token-to-gitcoin)
-- [Legal](#legal)
-- [License](#license)
-
-### On Github
-
-[Star](https://github.com/gitcoinco/web/stargazers) and [watch](https://github.com/gitcoinco/web/watchers) this github repository to stay up to date, we're pushing new code several times per week!
-
-Check out the [CHANGELOG](https://docs.gitcoin.co/mk_changelog/) for details about recent changes to this repository.
-
-Also,
-
-* want to become a contributor ? Checkout our [guidelines](https://docs.gitcoin.co/mk_contributors/) & [styleguide](https://docs.gitcoin.co/mk_styleguide/).
-* [check out the gitcoinco organization-wide repo](https://github.com/gitcoinco/gitcoinco).
-* check out the open issues list, especially the [discussion](https://github.com/gitcoinco/web/issues?q=is%3Aissue+is%3Aopen+label%3Adiscussion) label and [easy-pickings](https://github.com/gitcoinco/web/issues?q=is%3Aissue+is%3Aopen+label%3Aeasy-pickings).
-
-### On Gitcoin
-
-[Check out the available open issues on Gitcoin](https://gitcoin.co/explorer/?q=https://github.com/gitcoinco/web).
+- [High Level flows](#high-level-flows)
+  * [Bounty](#bounty)
+  * [Tip or Kudos](#tip-or-kudos)
+- [Licenses](#Licenses)
 
 ## What is Gitcoin?
 
-Functionally, the app has several key features:
+Gitcoin is an open source bounties platform on the Ethereum blockchain. We facilitate a space that allows open source developers to get paid for their work contributing to open source projects and in return, the open source projects get exposure to a vast community of hard working developers they might not have had otherwise.
+
+## Functionally
```

Comment:
Could we make it `###` as it's a subcategory of `What is Gitcoin` ?

Reply:
done also Technically is sub of what's Gitcoin